### DISCUSSION TITLE
Big class

### DISCUSSION THREAD

**Discussion by hose1021** [tailwindlabs/tailwindcss#4488] [General]
What do you think about many classes in one tag?
For example if we will use `focus:` `hover:` `dark:` `xs-xl:` `before:` `after:` for one tag, it will be very big. Is it a problem?

**Comment by devntv** [comment]
> What do you think about many classes in one tag?
> For example if we will use `focus:` `hover:` `dark:` `xs-xl:` `before:` `after:` for one tag, it will be very big. Is it a problem?

You can use @apply
documentation: https://tailwindcss.com/docs/functions-and-directives#apply
